packages/jest-cli/src/TestRunner.js

```diff
@@ -379,6 +379,10 @@ class TestRunner {
   _setupReporters() {
     const config = this._config;
 
+    if (config.reporters) {
+      return this._addCustomReporters(config.reporters);
```

Comment:
So when you setup any custom reporter, you're getting rid of all default ones, like `DefaultReporter`, `CoverageReporter`, `SummaryReporter`? I'm not happy with that